domains/lego.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "matskugaming",
+           "email": "matti.ju.laaksonen@gmail.com",
+           "discord": "975721598785060895"
+        },
+    
+        "record": {
+            "TXT": "lego"
```

Comment:
TXT record alone doesn't do anything. Your site isn't being displayed anywhere. Also LEGO is a trademark, so this domain will not be released for abuse risk.